packages/state-manager/src/sagas/communities/saveCommunityMetadata/saveCommunityMetadata.saga.ts

```diff
@@ -22,4 +23,8 @@ export function* saveCommunityMetadataSaga(
       ownerCertificate: action.payload.ownerCertificate,
     })
   )
+
+  const community = yield* select(communitiesSelectors.currentCommunity)
```

Comment:
This is way better, although to avoid confusion I'd have rather emit `sendUnregisteredInfoMessage` action directly in `startConnection.saga.ts:261` and moved checking the presence of community.CA inside `sendUnregisteredInfoMessageSaga` (it wouldn't be my first shot to look for sending message from within saveCommunityMetadataSaga)

Reply:
Actually, we must depend on channels rather than community metadata replication event